copying.md

```diff
@@ -6,139 +6,139 @@ A copy of the license can be found in [legal/GPLV3](/legal/GPLv3).
+| Julius Michaelis            | jcaesar                     | gitter à liftm dawt de                           |
+| Katharina Bogad             | mistressofjellyfish         | delirium à hacked dawt xyz                       |
+| Oliver Fawcett-Griffiths    | ollyfg                      | olly à ollyfg dawt com                           |
+| Ross Murray                 | rossmurray                  | rm à egoorb dawt com                             |
+| Alexandre Arpin             | AlexandreArpin              | arpin dawt alexandre à gmail dawt com            |
+| Henry Snoek                 | snoek09                     | snoek09 à gmail dawt com                         |
+| Gabriel Scherer             | gasche                      | gasche dawt dylc à gmail dawt com                |
+| Austin Eyler                | awestin1                    | awestin1 à gmail dawt com                        |
+| Francisco Demartino         | franciscod                  | demartino dawt francisco à gmail dawt com        |
+| Peter Piwowarski            | oldlaptop                   | oldlaptop654 à aol dawt com                      |
+| Charles Pigott              | LordAro                     | charlespigott à googlemail dawt com              |
+| Andrew Eikum                | ColdPie1                    | coldpies à gmail dawt com                        |
+| Michael Sebastiyan          | BugExplorer                 | sebastiyan dawt michael à outlook dawt com       |
+| Adam Miartus                | miartad                     | adam dawt miartus à gmail dawt com               |
+| Benoît Legat                | blegat                      | benoit dawt legat à gmail dawt com               |
+| James Hagborg               | blucoat                     | jameshagborg à gmail dawt com                    |
+| Prashanth Jonnala           | jprashanth                  | prashanth dawt neo à gmail dawt com              |
+| Jonathan Remnant            | Jon0                        | jono4728 à gmail dawt com                        |
+| Sam Schetterer              | schets                      | samschet à gmail dawt com                        |
+| Georg Kilzer                | leper                       | leper à wildfiregames dawt com                   |
+| Florian Erler               | ethon                       | ethon à ethon dawt cc                            |
+| Michał Janiszewski          | janisozaur                  | janisozaur+openage à gmail dawt com              |
+| Lautaro Nahuel De León      | lndl                        | laudleon à gmail dawt com                        |
+| Robin Kreis                 | rkreis                      | r dawt kreis à uni-bremen dawt de                |
+| Shion Ryuu                  | shion                       | shionryuu à outlook dawt com                     |
+| Jonas Borchelt              | riotjones                   | jonas dawt borchelt à connected dawt link        |
```

Comment:
This will make kevin complain about the old commits. You should add your new mail to `.mailmap` instead.

Reply:
check